packages/common/src/utils.ts

```diff
@@ -93,6 +96,33 @@ function parseGethParams(json: any) {
     throw new Error('nonzero terminal total difficulty is not supported')
   }
 
+  let customHardforks: HardforksDict | undefined = undefined;
+  if (config.blobSchedule !== undefined) {
+    customHardforks = {};
+    const blobGasPerBlob = 131072;
```

Comment:
Ideally, we do not hard-code this constant and read it from `blobGasPerBlob` (= 2**17 / 131072)